scripts/fdbshow

```diff
@@ -163,11 +165,25 @@ class FdbShow(object):
             self.bridge_mac_list = [fdb for fdb in self.bridge_mac_list[s_index:]
                                     if fdb[2] == port]
 
+        if address is not None:
+            address = address.upper()
+            s_index = self.get_iter_index(address, 1)
+            self.bridge_mac_list = [fdb for fdb in self.bridge_mac_list[s_index:]
+                                    if fdb[1] == address]
+
+        if entry_type is not None:
+            entry_type = entry_type.capitalize()
+            s_index = self.get_iter_index(entry_type, 3)
+            self.bridge_mac_list = [fdb for fdb in self.bridge_mac_list[s_index:]
+                                   if fdb[3] == entry_type]
```

Comment:
get_iter_index() will sort the bridge_mac_list.
If input has vlan, port, address and entry_type, bridge_mac_list will be sorted for 4 times.
How about use single loop to filter mac address? For example:


```
for fdb in self.bridge_mac_list:
    if (vlan is None or fdb[0] == vlan) and
        (port is None or fdb[2] == port) and
        (address is None or fdb[1] == address) and
        (entry_type is None or fdb[3] == entry_type):
```

Reply:
Addressed these changes in latest commit.